launch.py

```diff
@@ -113,14 +115,15 @@ def load_builtin_nlprocessor_models():
     initnlprocessor_project = "docker.greenkeytech.com/nlpmodelcontroller"
     initnlprocessor_image = f"{initnlprocessor_project}:{initnlprocessor_tag}"
     pull_image(initnlprocessor_image)
-    get_docker_client().containers.run(
+    initcontainer = get_docker_client().containers.run(
         initnlprocessor_image,
-        auto_remove=True,
-        volumes={builtin_models.name: {
-            "bind": "/models/transformers",
-            "mode": "rw"
-        }},
+        auto_remove=False,
+        detach=True,
+        volumes={builtin_models.name: {"bind": "/models/transformers", "mode": "rw"}},
     )
+    initcontainer.start()
+    initcontainer.wait()
```

Comment:
How long is waiting? Should we have log messages to alert users of what we're waiting for?

Reply:
This is a blocking wait until the container exits
https://docker-py.readthedocs.io/en/stable/containers.html#docker.models.containers.Container.wait

Reply:
Have you notices this takes a long time? If it takes a long time, I think we should add a log message alerting users who're waiting for the container to exit.

Reply:
It takes as long as the file copies take, which is a few seconds. The really long steps are the docker pull for init containers and the discovery startup. I'm personally holding off on adding further logging until we capture and prune the logging of docker compose, which overwhelms everything else.

Reply:
If the logs will overrun, then no need to add them.